kymatio/scattering2d/backend/jax_backend.py

```diff
@@ -0,0 +1,119 @@
+# Authors: Edouard Oyallon, Sergey Zagoruyko, Muawiz Chaudhary
+
+import jax.numpy as np
+from collections import namedtuple
+import scipy.fftpack
+
+BACKEND_NAME = 'jax'
+
+
+from ...backend.jax_backend import modulus, cdgmm, complex_check, real_check
+
+
+class Pad(object):
+    def __init__(self, pad_size, input_size, pre_pad=False):
+        """
+            Padding which allows to simultaneously pad in a reflection fashion
+            and map to complex.
+
+            Parameters
+            ----------
+            pad_size : list of 4 integers
+                size of padding to apply.
+            input_size : list of 2 integers
```

Comment:
what do we need this arg for? It seems to get completely lost. Do other backends need it?

Reply:
I believe torch needs it. https://github.com/kymatio/kymatio/blob/dev/kymatio/scattering2d/backend/torch_backend.py#L76

Reply:
I don't actually understand why it can't get that out of the batch. it's not like it's precomputing anything